usr/share/rear/layout/save/GNU/Linux/220_lvm_layout.sh

```diff
@@ -82,6 +82,9 @@ local lvs_exit_code
         # Skip lines that are not describing physical devices
         # i.e. lines where pdev does not start with a leading / character:
         test "${pdev#/}" = "$pdev" && continue
+        if [[ "${pdev}" =~ '/dev/loop' ]]; then
+            continue
```

Comment:
@david-hill 
from plain looking at the code
I wonder why you need a compund command `[[ expression ]]`
with the '=~' operator when the string to the right
is no regular expression?

I assume it is meant to test if $pdev starts with /dev/loop
so wouldn't
```
[[ $pdev = /dev/loop/* ]] && continue
```
be more correct because =~ '/dev/loop'
matches when '/dev/loop' is any substring in $pdev
and more clear code?

Reply:
I used your fist suggested method because it's easier to read.    I don't think /dev/loop/dev/loop0 can exist so even if my regular expression wasn't looking for ^/dev/loop.*$ it would still have worked properly.

Reply:
@david-hill 
I also don't think e.g. /dev/loop/dev/loop0 will ever exist
in practice so I agree it would have worked in practice.
My point is that your code was less clear when reading it
so at some future time (perhaps years later) other people
who then may like to adapt/enhance/fix something there
may needlessly wonder what the subtle reason could be
why substring match has to be used there ;-)
cf. "Code must be easy to understand (answer the WHY)" in
https://github.com/rear/rear/wiki/Coding-Style